src/components/photography/slider.js

```diff
@@ -16,149 +122,153 @@ class Carousel extends Component {
-    delete props.indicator
-    delete props.transitionDuration
-    delete props.transitionTimingFunction
-    delete props.slideWillChange
-    delete props.slideDidChange
-    delete props.initialSlide
-    const { slide, sliding, dragging, offset } = this.state
-    const slides = Children.map(children, (child) => React.cloneElement(child, { key: child.key + '_clone' }))
-    const count = Children.count(children)
-    const enabled = count > 1
-    const goPrevSlide = this.changeSlide.bind(this, slide - 1)
-    const goNextSlide = this.changeSlide.bind(this, slide + 1)
-    const slideStyle = {
-      flexBasis: '100%',
-      flexShrink: 0
-    }
+    const {
+      className,
+      transitionDuration,
+      transitionTimingFunction
+    } = this.props
+    const { currentSlide, isSliding, offset } = this.state
+    const goPrevSlide = this.changeSlide.bind(this, currentSlide - 1)
+    const goNextSlide = this.changeSlide.bind(this, currentSlide + 1)
+
     return (
-      <div {...props} style={Object.assign({}, props.style, {
-        position: 'relative',
-        overflowX: 'hidden',
-        touchAction: 'pan-y pinch-zoom'
-      })}>
-        <ul style={{
-          listStyleType: 'none',
-          padding: 0,
-          margin: 0,
-          display: 'flex',
-          transitionProperty: sliding ? 'transform' : 'none',
-          transform: enabled ? (offset !== 0 ? 'translateX(calc(' + (offset * 1) + 'px - ' + slide * 100 + '%))' : 'translateX(-' + slide * 100 + '%)') : null,
-          transitionDuration,
-          transitionTimingFunction,
-          contain: 'layout',
-          willChange: 'transform'
-        }} {...this.events}>
-          {enabled && Children.map(slides.slice(-1).concat(children, slides.slice(0, 1)),
-            (item, index) => <li aria-current={slide === index} style={slideStyle}>{item}</li>) || <li>{children}</li>
-          }
-        </ul>
-        {enabled && indicator && <ol>
-          {Children.map(children, (item, index) =>
-            <li aria-current={slide === index + 1} onClick={this.changeSlide.bind(this, index + 1)}>
-              {index + 1}
-            </li>
-          )}
-        </ol>}
-        {enabled && switcher && <div>
-          <button type="button" className="prev" onClick={goPrevSlide}></button>
-          <button type="button" className="next" onClick={goNextSlide}></button>
-        </div>}
+      <div className={className}>
+        <SlidesContainer>
+          <Slides
+            currentSlide={currentSlide}
+            isSliding={isSliding}
+            offset={offset}
+            transitionDuration={transitionDuration}
+            transitionTimingFunction={transitionTimingFunction}
+            {...this.events}
+          >
+            {this._renderSlides()}
+          </Slides>
+          <PrevBtn type="button" className="prev" onClick={goPrevSlide}></PrevBtn>
+          <NextBtn type="button" className="next" onClick={goNextSlide}></NextBtn>
```

Comment:
In Chrome, `type="button"` will adopt `-webkit-appearance: button` style,  and which hides overlay a default button on svgs.